angular/Listbox/Appearance-and-Styling.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Syncfusion ListBox Appearance-and-Styling
-description: appearance and styling
+title: Appearance-and-Styling | ListBox | Angular | Syncfusion
+description: Learn here about Appearance and Styling in Syncfusion Essential Angular ListBox Control, its elements, and more.
```

Comment:
Control name, Platform name, Syncfusion - These terms first letter should be capital.
Others must in small case. Change as per this. Check and replace in all places.

Reply:
Corrected